doc/usage.md

```diff
@@ -570,6 +570,12 @@ max_connections
 current_connections
 :   Current number of connections for this database.
```

Comment:
```suggestion
:   Current number of server connections for this database.
```

Reply:
and let's make the same change for `max_connections` above (I can't comment on that line because it's outside the diff according to github...)

Reply:
Done